src/main/java/com/whytowait/domain/models/HashedPassword.java

```diff
@@ -11,15 +12,19 @@
 @Entity
 @AllArgsConstructor
 @NoArgsConstructor
+@Table(name = "hashed_password")
 public class HashedPassword {
+
     @Id
-    @GeneratedValue(strategy = GenerationType.AUTO)
-    private UUID Id;
+    @GeneratedValue
+    private UUID id;
 
-    @Column(nullable = false)
-    private String HashedPassword;
+    @NotNull
+    @OneToOne(fetch = FetchType.LAZY)
+    @JoinColumn(name = "user_id", referencedColumnName = "id", nullable = false)
```

Comment:
Here when i request i will get a complete user right ? , shouldn't we just send userId.

Reply:
we will use join queries, removed relationships